crates/node-bindings/src/helpers/js_callable/local_functions.rs

```diff
@@ -0,0 +1,47 @@
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::rc::Rc;
+
+use super::JsValue;
+use napi::bindgen_prelude::FromNapiValue;
+use napi::Env;
+use napi::JsFunction;
+
+thread_local! {
+  /// Storage for napi JavaScript functions on the local thread
+  static LOCAL_FUNCTIONS: (RefCell<usize>, RefCell<HashMap<usize, Rc<JsValue>>>) = Default::default();
```

Comment:
V8 may GC this value we can't hold it;

In order to keep it we need to use - https://napi.rs/docs/compat-mode/concepts/ref

Reply:
Removed